### DISCUSSION TITLE
Should we change dialog button positioning?

### DISCUSSION THREAD

**Discussion by matthias-ronge** [kitodo/kitodo-production#4504] [General]
Buttons in dialogs are currently right-aligned and use two different colors. Both is inefficient to work with, as [eye-tracking analysis reveals](https://www.lukew.com/ff/entry.asp?571). The article shows that putting buttons to the left side, with the primary button first, and using the same color for both buttons is more efficient to work with.

Should we change dialog button positioning?

**Comment by solth** [comment]
On the one hand, it might be time to evaluate some design decisions made during the upgrade to Kitodo.Production 3, on the other hand many users might have become used to the current layout by now and would then be confused by such fundamental adjustments to the design at this stage.
@kaimertens, @finck what do you think? I would be interested to read your thoughts about this.

**Comment by kaimertens** [comment]
We take a look at the frontend (@solth and me). The principle of "icons arranged on the right" has been implemented conclusively. This affects the "dashboard", modal windows and window frames. If we want to change this principle, we have to revise all these points. In my opinion, there is no quick solution.
